.travis.yml

```diff
@@ -193,17 +195,21 @@ matrix:
     - env:
         - DESC="spell checker"
         - CMD="./.ci/test-spelling-unknown-words.sh"
+        - FORCE_CI = true
 
 script:
   - SKIP_FILES=".github|codeship-*|buddy.yml|appveyor.yml|circleci|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
-  - SKIP_CI=$(if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then echo false; else echo true; fi;)
+  - SKIP_CI=$(if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 && -z ${FORCE_CI+x} ]]; then echo false; else echo true; fi;)
   - echo "SKIP_CI="$SKIP_CI
+  - echo "FORCE_CI="$FORCE_CI
   - |
     set -e
     if [[ $SKIP_CI == 'false' ]];
```

Comment:
if (FORCE_CI) {
 eval ...
} esle if (SKIP_BY_FILES = false ) {
 eval ...
} esle {
 skip
}

Reply:
@romani I went the way I did because the old `SKIP_CI` is used in 3 different places. 
This line here is a simple single if statement which would have been an easy inclusion.
The other 2 are complicated if statements which would require something like `&& (SKIP_CI == false || FORCE_CI)`.
Example: `if [[ -n $CMD_AFTER_SUCCESS && $SKIP_CI == 'false' ]];`

I specifically merged `FORCE_CI` into `SKIP_CI` to avoid this as it would make those conditions too complex.